wiki/CSS/style.css

```diff
@@ -151,20 +151,25 @@ header {
 
 header > .navbar-nav {
   text-align: center;
-  margin-top: 16px;
 }
 
 /* Styling for upper nav */
 
 nav {
+  margin-top: 16px !important;
   transition: all 1s !important;
   -webkit-transition: top 0.5s; /* Safari */
```

Comment:
can you include the other versions of transition? like -o-transition and -moz-transition?